templates/web-desktop/index.ejs

```diff
@@ -24,7 +24,7 @@
 
     <div id="GameDiv" cc_exact_fit_screen="false" style="width: <%= previewWidth %>px; height: <%= previewHeight %>px;">
       <div id="Cocos3dGameContainer">
-        <canvas id="GameCanvas" width="<%= previewWidth %>" height="<%= previewHeight %>"></canvas>
+        <canvas id="GameCanvas" width="<%= previewWidth %>" height="<%= previewHeight %>" tabindex="99"></canvas>
```

Comment:
why change tabIndex to 99 ?

Reply:
![image](https://user-images.githubusercontent.com/15109777/176408532-25079b75-b215-4aaa-a1c9-f1169fb079a0.png)
keep same logic

Reply:
ok